### PR TITLE
Addressed re:invent feedbacks

### PR DESCRIPTION
**Pull request by lanazhang**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content-moderation/02-video-moderation/01-content-moderation-video.ipynb [hunk 1]
```diff
@@ -261,35 +249,63 @@
...
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Run the below file to print out all the detected texts. Click on the item to navigate to the above video's timestamp."
+    "# Step 4: Moderate audio in video <a id=\"step4\"></a>\n",
+    "\n",
+    "We have moderated the video and extracted text from the video using Rekognition. Now, we will transcribe the audio to text using Amazon Transcribe.\n",
+    "\n",
+    "Call Transcribe **StartTranscriptionJob** API to trascribe the audio to text. Amazon Transcribe StartTranscriptionJob is an asynchronize API that will start a job managed by Transcribe. We will then use iteration logic to heart-beat the get_transcription_job API to check job status until the job completes."
```
Comment (matteofigus) [inline content-moderation/02-video-moderation/01-content-moderation-video.ipynb]:
```suggestion
    "Call Transcribe **StartTranscriptionJob** API to transcribe the audio to text. Amazon Transcribe StartTranscriptionJob is an asynchronous API that will start a job managed by Transcribe. We will then use iteration logic to heart-beat the **GetTranscriptionJob** API to check job status until the job completes."
```
Typo + consistency on API namings.

File: content-moderation/02-video-moderation/01-content-moderation-video.ipynb [hunk 2]
```diff
@@ -15,26 +15,28 @@
...
     "\n",
-    "In this tutorial, we will show you how to use Amazon Rekognition Moderation API to moderate stored videos. And how to moderate text in a video using Amazon Rekognition and Amazon Comprehend."
+    "This tutorial will show you how to use Amazon Rekognition Moderation API to moderate stored videos. And how to extract text in a video using Amazon Rekognition for Text Moderation. And how to transcribe the audio of the video into text for Text Moderation."
```
Comment (matteofigus) [inline content-moderation/02-video-moderation/01-content-moderation-video.ipynb]:
```suggestion
    "This tutorial will show you how to use Amazon Rekognition Moderation API to moderate stored videos, how to extract text in a video using Amazon Rekognition for Text Moderation, and how to transcribe the audio of the video into text for Text Moderation."
```
nit: don't start a sentence with a conjunction to improve clarity.

File: content-moderation/02-video-moderation/01-content-moderation-video.ipynb [hunk 3]
```diff
@@ -205,7 +193,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Text detection with Amazon Rekognition Video is an asynchronous operation. You start text detection by calling **StartTextDetection** which returns a job identifier (JobId ) When the text detection operation finishes, Amazon Rekognition publishes a completion status to the Amazon Simple Notification Service topic registered in the initial call to **StartTextDetection**. To get the results of the text detection operation, first check that the status value published to the Amazon SNS topic is SUCCEEDED . if so, call **GetTextDetection** and pass the job identifier (JobId ) from the initial call of **StartLabelDetection**.\n",
+    "Like the Video moderation API, text detection with Amazon Rekognition Video is also an asynchronous operation. You start text detection by calling StartTextDetection, which returns a job identifier (JobId). Then we will periodically poll the job status by calling GetTextDetection API. Until the job status becomes SUCCEED.\n",
```
Comment (matteofigus) [inline content-moderation/02-video-moderation/01-content-moderation-video.ipynb]:
```suggestion
    "Similarly to the Video moderation API, text detection with Amazon Rekognition Video is also an asynchronous operation. You start text detection by calling StartTextDetection, which returns a job identifier (JobId). Then we will periodically poll the job status by calling GetTextDetection API. Until the job status becomes SUCCEED.\n",
```

File: content-moderation/03-audio-moderation/01-content-moderation-audio-transcribe-with-custom-vocabulary.ipynb [hunk 4]
```diff
@@ -99,7 +100,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Call Transcribe **StartTranscriptionJob** API to trascribe the audio to text. Amazon Transcribe StartTranscriptionJob is an asynchronize API that will start a job managed by Transcribe. We will receive a job_id back when we start the job. Then use iteration logic to heart-beat the get_transcription_job API to check job status until the job completes."
+    "Call Transcribe **StartTranscriptionJob** API to trascribe the audio to text. Amazon Transcribe StartTranscriptionJob is an asynchronize API that will start a job managed by Transcribe. We will then use iteration logic to heart-beat the get_transcription_job API to check job status until the job completes."
```
Comment (matteofigus) [inline content-moderation/03-audio-moderation/01-content-moderation-audio-transcribe-with-custom-vocabulary.ipynb]:
```suggestion
    "Call Transcribe **StartTranscriptionJob** API to transcribe the audio to text. Amazon Transcribe StartTranscriptionJob is an asynchronous API that will start a job managed by Transcribe. We will then use iteration logic to heart-beat the **GetTranscriptionJob** API to check job status until the job completes."
```

File: content-moderation/02-video-moderation/01-content-moderation-video.ipynb [hunk 5]
```diff
@@ -261,35 +249,63 @@
...
-    "display(HTML(video_tag))"
+    "display(HTML(video_tag))\n",
+    "display(HTML(text_html))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After extracting texts from the video, we can then moderate the texts using Amazon Comprehend. This lab will not cover the text moderation logic. You can refer to the 04-text-moderation module for more examples."
```
Comment (matteofigus) [inline content-moderation/02-video-moderation/01-content-moderation-video.ipynb]:
```suggestion
    "After extracting text from the video, we can then moderate it using Amazon Comprehend. This lab will not cover the text moderation logic. You can refer to the 04-text-moderation module for more examples."
```

File: content-moderation/02-video-moderation/01-content-moderation-video.ipynb [hunk 6]
```diff
@@ -298,28 +314,26 @@
...
+    "original = json.loads(s3_clientdata)\n",
+    "output_transcript = original[\"results\"][\"transcripts\"]\n",
+    "print(output_transcript)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After transcribing texts from the audio of the video, we can then moderate the texts using Amazon Comprehend. This lab will not cover the text moderation logic. You can refer to labs in the 04-text-moderation module for more examples."
```
Comment (matteofigus) [inline content-moderation/02-video-moderation/01-content-moderation-video.ipynb]:
```suggestion
    "After transcribing text from the audio of the video, we can then moderate it using Amazon Comprehend. This lab will not cover the text moderation logic. You can refer to labs in the 04-text-moderation module for more examples."
```

**Review by matteofigus — CHANGES_REQUESTED** [review]
A couple of typos and suggestions.

File: content-moderation/02-video-moderation/01-content-moderation-video.ipynb [hunk 7]
```diff
@@ -15,26 +15,28 @@
...
     "\n",
-    "In this tutorial, we will show you how to use Amazon Rekognition Moderation API to moderate stored videos. And how to moderate text in a video using Amazon Rekognition and Amazon Comprehend."
+    "This tutorial will show you how to use Amazon Rekognition Moderation API to moderate stored videos, how to extract text in a video using Amazon Rekognition for Text Moderation, and how to transcribe the audio of the video into text for Text Moderation."
```
Comment (ctd) [inline content-moderation/02-video-moderation/01-content-moderation-video.ipynb]:
```suggestion
    "This tutorial will show you how to use Amazon Rekognition Moderation API to moderate stored videos, how to extract text from a video using Amazon Rekognition for Text Moderation, and how to transcribe the audio of the video into text for Text Moderation."
```

File: content-moderation/02-video-moderation/01-content-moderation-video.ipynb [hunk 8]
```diff
@@ -78,14 +80,16 @@
...
+    "comprehend = boto3.client('comprehend', region_name=region)\n",
+    "transcribe=boto3.client('transcribe', region_name=region)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For this lab, we will moderate a demo video containing a few typical inappropriate scenes, such as alcohol, tobacco, gambling, and suggestive. The video locates at `datasets/moderation-video.mp4`.\n",
+    "For this lab, we will moderate a demo video containing typical inappropriate scenes, such as alcohol, tobacco, and suggestive. The video also has text and audio containing inappropriate information, such as profanity. You can find the video at `datasets/moderation-video.mp4`.\n",
```
Comment (ctd) [inline content-moderation/02-video-moderation/01-content-moderation-video.ipynb:91]: What is meant by "suggestive"?

My best guess is "sexually suggestive", but could there be terms that are more objective such as "nudity" or "sexual activity"? (I'm taking inspiration from the likes of https://www.bbfc.co.uk/rating/12 and https://www.filmratings.com/RatingsGuide).

At the least, "suggestive" is ambiguous and should be elaborated upon.

File: content-moderation/02-video-moderation/01-content-moderation-video.ipynb [hunk 9]
```diff
@@ -261,35 +249,63 @@
...
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Run the below file to print out all the detected texts. Click on the item to navigate to the above video's timestamp."
+    "# Step 4: Moderate audio in video <a id=\"step4\"></a>\n",
+    "\n",
+    "We have moderated the video and extracted text from the video using Rekognition. Now, we will transcribe the audio to text using Amazon Transcribe.\n",
+    "\n",
+    "Call Transcribe **StartTranscriptionJob** API to transcribe the audio to text. Amazon Transcribe StartTranscriptionJob is an asynchronous API that will start a job managed by Transcribe. We will then use iteration logic to heart-beat the **GetTranscriptionJob** API to check job status until the job completes."
```
Comment (ctd) [inline content-moderation/02-video-moderation/01-content-moderation-video.ipynb]:
```suggestion
    "Call Transcribe **StartTranscriptionJob** API to transcribe the audio to text. Amazon Transcribe StartTranscriptionJob is an asynchronous API that will start a job managed by Transcribe. We will then call the **GetTranscriptionJob** API to check job status until the job completes."
```

"iteration logic" and "heart-beat" are both potentially jargon-ish and perhaps unnecessarily complex to describe what is happening here. Simplifying should make it easier for international audiences to follow the content whilst still conveying the same meaning.

File: content-moderation/03-audio-moderation/01-content-moderation-audio-transcribe-with-custom-vocabulary.ipynb [hunk 10]
```diff
@@ -99,7 +100,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Call Transcribe **StartTranscriptionJob** API to trascribe the audio to text. Amazon Transcribe StartTranscriptionJob is an asynchronize API that will start a job managed by Transcribe. We will receive a job_id back when we start the job. Then use iteration logic to heart-beat the get_transcription_job API to check job status until the job completes."
+    "Call Transcribe **StartTranscriptionJob** API to transcribe the audio to text. Amazon Transcribe StartTranscriptionJob is an asynchronous API that will start a job managed by Transcribe. We will then use iteration logic to heart-beat the **GetTranscriptionJob** API to check job status until the job completes."
```
Comment (ctd) [inline content-moderation/03-audio-moderation/01-content-moderation-audio-transcribe-with-custom-vocabulary.ipynb]:
```suggestion
    "Call Transcribe **StartTranscriptionJob** API to transcribe the audio to text. Amazon Transcribe StartTranscriptionJob is an asynchronous API that will start a job managed by Transcribe. We will then call the **GetTranscriptionJob** API to check the job status until the job completes."
```

**Review by ctd — CHANGES_REQUESTED** [review]
A few small in-line suggestions.

I am quite concerned about the phrase "You can ignore the warnings/errors." that is introduced in several places. I assume that you mean there are certain warnings/errors that are expected, but the phrase gives the reader permission to ignore **all** errors. Like, what if the error is "nope i can't install any of your dependencies" - something like that can't be ignored because everything that follows in the workshop will fail.

Could you perhaps extrapolate this some way to say that **some** warnings/errors are expected, and give the reader a way of determining what their output should look like? (e.g. an example of the expected output in text or screenshot form).

File: content-moderation/02-video-moderation/01-content-moderation-video.ipynb (hunk 8 above)
Comment (lanazhang) [reply content-moderation/02-video-moderation/01-content-moderation-video.ipynb:91]: "Suggestive" is a Rekognition predefined Content Moderation category.

**Comment by lanazhang** [comment]
Chris, about the pip error and warning. I updated the instruction to:
You can ignore the warnings/errors related to Python virtual environment recommendations or pip dependencies. The update steps are optional if you run the lab in a SageMaker Studio Juypter Notebook.

**Review by ctd — APPROVED** [review]
Thanks for the updates! LGTM now.

**Review by matteofigus — APPROVED** [review]

**Pull request merged**